blocks/subscriptions-block/components/useOrder.jsx

```diff
@@ -0,0 +1,152 @@
+import { useState, useEffect } from "react";
+
+import { useSales } from "@wpmedia/arc-themes-components";
+import { ARCXP_CAMPAIGN } from "./OfferToProductList";
+import useOffer from "./useOffer";
+
+const useOrder = (orderNumber) => {
+	const { Sales } = useSales();
+
+	const [order, setOrder] = useState();
+	const [orderDetails, setOrderDetails] = useState();
+	const [cart, setCart] = useState();
+	const [cartDetails, setCartDetails] = useState();
+	const [error, setError] = useState();
+
+	const [campaignName, setCampaignName] = useState();
+
+	const { offer } = useOffer({
+		campaignCode: campaignName,
+	});
+
+	useEffect(() => {
+		const getCart = async () => {
+			try {
+				const currentCart = await Sales.getCart();
+				if(currentCart?.items?.length){
+					setCart(currentCart);
+				}
+			} catch (e) {
+				setError(e);
+			}
+		};
+		getCart();
+
+		const campaignNameStored = localStorage.getItem(ARCXP_CAMPAIGN);
+		setCampaignName(campaignNameStored);
+		// eslint-disable-next-line
```

Comment:
Just curious but have you checked with Vito on what's the process for ignoring one off linting errors like this?